packages/eslint-config/package.json

```diff
@@ -32,31 +32,20 @@
     "@10up/babel-preset-default": "^2.0.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/eslint-parser": "^7.16.3",
-    "@wordpress/eslint-plugin": "^11.0.0",
     "eslint": "^8.0.0",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jest": "^25.2.4",
-    "eslint-plugin-jsdoc": "^37.0.3",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.27.0",
-    "eslint-plugin-react-hooks": "^4.3.0",
     "prettier": "^2.4.1"
   },
   "peerDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/eslint-parser": "^7.16.3",
+	"@wordpress/eslint-plugin": "^12.0.0",
```

Comment:
Can we make this an optional dependency? We don't need it for non-wp projects and I'd rather not install WP specific things for non-wp projects;

Reply:
We can, but we install JSX, React and React and React Hooks dependencies even if we don't use them. 

I think being inconsistent and also having mixed approaches depending on what you need to use felt a bit odd. This is a dev dependency in any case. 

Let me know your thoughts !

Reply:
We don't install react, that's responsibility of the consumer. When using toolkit in WP context, the WP bundled react will be used.

toolkit already install `@wordpress/eslint-plugin` and everyone using eslint-config for WP would prob just use toolkit instead and would almost never install eslint-config directly, so leaving our eslint-config with no wp-specific stuff by default sounds reasonable to me.

Reply:
What I mean is that JSX and Hooks and React eslint packages are installed

Reply:
IMHO it's a bit odd to be asked to install other packages. Adding to that is the fact that different linting scenarios are treated differently which is why I did that.